integration-test/trace-debugger/build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+repositories {
+    mavenCentral()
+    maven { url = uri("https://repo.gradle.org/gradle/libs-releases/") }
+}
+
+sourceSets {
+    main {
+        java.srcDirs("src/main")
+
+        resources {
+            srcDir("src/main/resources")
+        }
+    }
+
+    dependencies {
+        val slf4jVersion: String by project
+
+        implementation(project(":integration-test:common"))
+        implementation("org.slf4j:slf4j-simple:$slf4jVersion")
+    }
+}
+
+tasks {
+    processResources {
+        duplicatesStrategy = DuplicatesStrategy.INCLUDE
+    }
+
+    registerTraceAgentIntegrationTestsPrerequisites()
+
+    val traceDebuggerIntegrationTest = register<Test>("traceDebuggerIntegrationTest") {
+        configureJvmTestCommon(project)
+        group = "verification"
+        include("org/jetbrains/test/trace/debugger/*")
+
+        testClassesDirs = sourceSets["main"].output.classesDirs
+        classpath = sourceSets["main"].runtimeClasspath
```

Comment:
I think previously it was for rootProject's main source set. Do we still need it?